eng/scripts/Language-Settings.ps1

```diff
@@ -206,6 +206,11 @@ function Get-DocsMsPackageName($packageName, $packageVersion) {
   return "$(Get-PackageNameFromDocsMsConfig $packageName)@$packageVersion"
 }
 
+$PackageExclusions = @{ 
+  '@azure/identity-vscode' = $true; # Fails type2docfx execution https://github.com/Azure/azure-sdk-for-js/issues/16303
```

Comment:
Lets make the value the actual comment string with the reason so we can dump it out in the logs when we are excluding this package.